en/services/camera.md

```diff
@@ -261,6 +261,46 @@ The steps are:
 > **Note** If your camera only provides video streaming and nothing else (no camera features), the [CAMERA_CAP_FLAGS_HAS_VIDEO_STREAM](../messages/common.md#CAMERA_CAP_FLAGS_HAS_VIDEO_STREAM) flag is the only flag you need to set.
   The GCS will then provide video streaming support and skip camera control.
 
+
+### Camera Tracking
+
+A camera may support tracking a point and/or a rectangle.
+Information about the tracked point is streamed during tracking, and may be passed to a gimbal in order to move the camera to track the target (or control vehicle attitude to track the target).
```

Comment:
Pedantic, but it is only streamed if you explicitly make it so. So "can", not "is". All tracking does is update the internal record with the tracking info, but does not send it out. I'll expand on this below.

```suggestion
Information about the tracked point can be streamed during tracking, and may be passed to a gimbal in order to move the camera to track the target (or control vehicle attitude to track the target).
```